distrobox

```diff
@@ -51,6 +49,19 @@ if [ $# -eq 0 ]; then
 	exit
 fi
 
+# Handle 'help' here, before setting 'errexit', so we have a chance
+# to show our help if the man command is there but fails.
```

Comment:
Instead of moving it before, how about using

`man distrobox || show_help`

This will ensure the show help after to be executed if man fails

Reply:
Sure. And done. :-)